README.md

```diff
@@ -1,73 +1,45 @@
-# Node.js (React, Express) Project Starter ✅
+# Cool Husky Games
 
-A simple template repository with a React frontend and Express backend for quickly starting new full-stack projects.
+## Uploading your game files to the website
 
-# Notes for deploying to AWS
-The <script> tags in the head of the `public/index.html` and `public/404.html` files need to be removed, as they are GitHub pages hack specific.
+1. Run pygbag to compile your pygame-ce file and assets for the website. When done, the output folder should look like [this](https://drive.google.com/drive/folders/15yVl406Rvm_zXHZ0qWUeB_uFUsEN8hYQ?usp=drive_link).
```

Comment:
So once they run pygbag and compule their pygame-ce file, they'll get a folder like the one you linked?

Reply:
Yes, everything should be like that, I didn't include Pygbag instructions because I assumed that the other teams knew how to do it. Do you want a walkthrough from the other teams?